e2e/tests/comparison-diff-view/comparison-diff-view.spec.ts

```diff
@@ -1,6 +1,6 @@
 import { expect, test } from '../../fixtures';
 
-test.describe('Smoke tests', () => {
+test.describe.skip('Smoke tests', () => {
```

Comment:
Skipping for now, we will remove them in https://github.com/grafana/explore-profiles/pull/143